coral-common/src/main/java/com/linkedin/coral/common/HiveCommonRelBuilder.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.common;
+
+import org.apache.calcite.plan.Context;
+import org.apache.calcite.plan.RelOptCluster;
+import org.apache.calcite.plan.RelOptSchema;
+import org.apache.calcite.rex.RexBuilder;
+import org.apache.calcite.tools.FrameworkConfig;
+import org.apache.calcite.tools.Frameworks;
+import org.apache.calcite.tools.RelBuilder;
+
+
+public class HiveCommonRelBuilder extends RelBuilder {
```

Comment:
Please check if this override can be introduced in `HiveRelConverter` class and if we can use that [here ](https://github.com/linkedin/coral/pull/449/files#diff-525c66fa9dca629a4d819ec31869600a6d3559ae0287d1c572b93f7fd2aff785R173)

Reply:
Yes, I have merged the code into `HiveRelConverter` and move it to `coral-common` module